testbed/eks/lib/addons/fluxv2.ts

```diff
@@ -0,0 +1,110 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import * as yaml from 'js-yaml';
+import * as request from 'sync-request';
+
+export interface FluxV2Props extends cdk.StackProps {
+    cluster: eks.Cluster;
+    fluxRepos: Repository[];
+    secretName: string;
+    fluxVersion: string;
+}
+export class Repository {
+    repoUrl: string;
+    repoBranch: string;
+    repoPath: string;
+
+    constructor(repoUrl: string, repoBranch?: string, repoPath?: string) {
```

Comment:
Instead of passing these variables as strings, which can introduce readability challenges from the caller side, use an options class pattern.

e.g. 
```
new Repository("foo", "bar", "baz") # <-- What are these variables?
```
vs 
```
new Repository({
  url: "...",
  branch: "...",
})
```

export class RepositoryOptions{
  branch: string
  path: string
  url: string
}

Reply:
Yup, makes sense. Similar to what I did for top level class Flux here - https://github.com/awslabs/kubernetes-iteration-toolkit/blob/testbed/testbed/eks/lib/testbed-stack.ts#L61-L65

https://github.com/awslabs/kubernetes-iteration-toolkit/blob/testbed/testbed/eks/lib/addons/fluxv2.ts#L6-L11